aiogram/dispatcher/filters/builtin.py

```diff
@@ -159,7 +160,7 @@ def text(self) -> str:
             if self.mentioned:
                 line += '@' + self.mention
             if self.args:
-                line += ' ' + self.args
+                line += ' ' + ' '.join(self.args)
```

Comment:
`self.args` is a `str`, you forgot to revert this
```suggestion
                line += ' ' + self.args
```